presto-docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -0,0 +1,30 @@
+===========
+SHOW STATS
+===========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW STATS FOR (query)
+
+Description
+-----------
+
+Returns stats for the results of ``query``.
+
+Stats are returned for each column, as well as an rollup across all columns (this
+rows has column_name of ``NULL``).
```

Comment:
my understanding is that the query needs to be of the shape `SELECT * FROM table` or `SELECT cols FROM table` with an optional filter on the partitioning keys e.g. `WHERE ds = 2018-09-07`.  If these restrictions on query shape are true, it would be great if you can document those too.